app/views/drafts/_form.html.erb

```diff
@@ -17,11 +17,17 @@
   <div class="col-6">
     <p class="eui-required-o">Indicates required field</p>
   </div>
+
+  <!-- TODO This shouldn't show on forms without multiple top level accordions? -->
```

Comment:
do you still need this comment?

Reply:
I'm not sure the answer to the question yet, sooo, maybe?

Reply:
So since you added the `expandable?` call, it seems that now this comment can be removed, right?